subprojects/dependency-management/src/integTest/groovy/org/gradle/api/artifacts/transform/ArtifactTransformBuildScanIntegrationTest.groovy

```diff
@@ -27,6 +32,7 @@ class ArtifactTransformBuildScanIntegrationTest extends AbstractIntegrationSpec
             include 'lib'
             include 'util'
         """
+        configureIncrementalArtifactTransformations(settingsFile)
```

Comment:
Shouldn't we check for incremental execution here?

Reply:
What do you mean with "check for incremental execution"? I need this to run the test with the feature flag enable/disabled.

Reply:
Got it.